### DISCUSSION TITLE
2024终于可以仅用1个css属性就实现垂直居中了!! 附:总结以往常用的实现垂直居中方法

### DISCUSSION THREAD

**Discussion by giscus** [pinky-pig/arvin#15] [General]
# 2024终于可以仅用1个css属性就实现垂直居中了!! 附:总结以往常用的实现垂直居中方法

原生CSS

http://localhost:1212/blogs/post/01-%E4%BB%85%E7%94%A81%E4%B8%AAcss%E5%B1%9E%E6%80%A7%E5%AE%9E%E7%8E%B0%E5%9E%82%E7%9B%B4%E5%B1%85%E4%B8%AD

**Comment by Mzz2022** [comment]
真有意思
